auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthJWTServiceActivator.java

```diff
@@ -40,6 +40,7 @@
 
   public static final String AUTH_JWT_IDENTIFIER = "AuthJwtService";
 
+  @SuppressWarnings("JdkObsolete")
```

Comment:
I'd replace by a `HashMap` (or `ConcurrentHashMap` if we're freaked out by concurrency, though I don't think it's an issue here).